index.js

```diff
@@ -8,6 +8,135 @@ function closeMenu() {
+    image: 'src/socials/github.svg',
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    live: 'https://www.google.com',
+    source: 'https://www.google.com',
+  },
+};
+
+function toggleModal(workId) {
+  const modal = document.getElementsByClassName('modal')[0];
+  const modalcontainer = document.getElementsByClassName('modal-container')[0];
+  function toggleBackground() {
+    const elements = [document.getElementsByClassName('introduction')[0], document.getElementsByTagName('main')[0], document.getElementsByClassName('aboutmeHeader')[0]];
+    Array.prototype.forEach.call(elements, (elem) => {
+      if (modal.style.display === 'flex') {
+        elem.style.background = '#C1C7D0';
+        elem.style.filter = 'blur(2px)';
+        document.body.style.background = '#C1C7D0';
+      } else {
+        elem.style.background = '';
+        elem.style.filter = 'blur(0)';
+        document.body.style.background = '';
+      }
+    });
+  }
+  function populateModal() {
+    function searchID(id) {
+      return document.getElementById(id);
+    }
+    const elemData = WORKOBJECT[workId];
+    const workName = searchID('workName');
+    const image = searchID('modal-image');
+    const techs = searchID('modal-technologies');
+    techs.innerHTML = '';
+    const desc = searchID('modal-description');
+    //const demoButton = searchID('live-demo');
+    //const sourceButton = searchID('source');
+
+    workName.innerHTML = elemData.name;
+
+    image.src = elemData.image;
+
+    elemData.technologies.forEach((elem) => {
+      const li = document.createElement('li');
+      li.appendChild(document.createTextNode(elem));
+      techs.appendChild(li);
+    });
+
+    desc.innerHTML = elemData.description;
+
+    // demoButton.href = elemData.live;
+    // sourceButton.href = elemData.source;
+  }
+
+  // background.style.background = "#C1C7D0";
+  if (modal.style.display !== 'flex') {
+    populateModal(workId);
+    modal.style.display = 'flex';
+    modalcontainer.style.display = 'flex';
+    modal.style.opacity = '1';
+    modalcontainer.style.opacity = '1';
+  } else {
+    modal.style.display = 'none';
+    modal.style.opacity = '';
+    modalcontainer.style.opacity = '';
+    modalcontainer.style.display = 'none';
+  }
+
+  toggleBackground();
+}
+
+function assignEventHandlers() {
+  const workcards = document.getElementsByClassName('workCard');
+  Array.prototype.forEach.call(workcards, (elem) => {
+    const workId = elem.id;
+
+    elem.getElementsByClassName('workInfo')[0].getElementsByTagName('button')[0].addEventListener('click', () => {
```

Comment:
- [ ] Your mobile menu is not functioning as expected. The menu overlay does not popup when the hamburger menu is clicked. Kindly ensure to improve upon this aspect

Reply:
* Kindly fix this as mentioned by the previous reviewer it still is doesn't look the same as the original design 
* Also i suggest you fix the error in the console so that your project looks professional and clean 
* [ORIGINAL]
![image](https://user-images.githubusercontent.com/52497246/131600218-ae51403a-0052-4363-93f8-5e74ffb8b356.png)
* [YOURS]
![image](https://user-images.githubusercontent.com/52497246/131600275-5f6e244f-9b1d-4d5a-adba-d283caa0aed9.png)